### ISSUE TITLE
Istio custom redis port is invalid

### ISSUE THREAD

**Issue by MicroOps-cn** [curiefense/curiefense#816]
Istio custom redis port is invalid, Although the port can be specified in the configuration, the corresponding environment variable is not actually called
In addition, for security, our redis server needs a password to link, but there seems to be no way to specify the redis password here

https://github.com/curiefense/curiefense/blob/4dc40461fc7b60c0861044b8cfe1cb0ca2f422e8/curiefense/curieproxy/rust/curiefense/src/redis.rs#L12-L17

**Comment by bartavelle** [comment]
Indeed! It would be nice to be able to specify port AND optional password.

**Comment by MicroOps-cn** [comment]
> Indeed! It would be nice to be able to specify port AND optional password.

👍，Can I customize the database (number)?

**Comment by bartavelle** [comment]
updated the PR

**Comment by MicroOps-cn** [comment]
> updated the PR
👍 thx
